storage_manager_new.py

```diff
@@ -5,46 +5,50 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.load_items_from_file()
 
     def load_items_from_file(self):
-        Item.instances = []
+        self.items = []
         File = open(Storage.database_file, 'r')
         for line in File.readlines():
             line = line.rstrip('\n')
-            Item.instance_from_string(line)
+            self.items.append(Item.create_instance_from_string(line))
 
     def add_item(self):
         # collect_new_item_data
         new_items_data = []
-        add_more = True
-        while add_more == True:
-            new_item = Interface.get_user_choice(
-                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).')
+        while True:
+            new_item = input(
+                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).\n')
             if new_item.lower() != 'menu':
                 new_items_data.append(new_item)
             else:
                 break
-            ask_for_more = Interface.get_user_choice(
-                'Would you like to add another item? Please type Y/N')
+            ask_for_more = input(
+                'Would you like to add another item? Please type Y/N\n')
             if ask_for_more.lower() == 'n':
-                add_more = False
+                break
         # create_item_instance and write to database_file
         for string in new_items_data:
             try:
-                data = string.split(', ')
-                quantity = int(data.pop(3))
-                string_data = ', '.join(data)
+                list_from_string = string.split(', ')
```

Comment:
List of what? We could use more descriptive name here. Naming is hard but also very important ;)